sample-queries.csv

```diff
@@ -172,6 +172,7 @@ Planner,POST,create a Planner task,https://graph.microsoft.com/v1.0/planner/task
 Planner,PATCH,update a Planner task,https://graph.microsoft.com/v1.0//planner/tasks/{task-id},Group.ReadWrite.All,This query requires a task id and value of @odata.etag for a selected task.  To find the ID of the task and @odata.etag you can run: GET https://graph.microsoft.com/v1.0/me/planner/tasks,https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/plannertask_update,AAD,"{
     ""title"": ""Updated task title""
 }",If-Match: {if-match},
+Planner,GET,details for Planner task,https://graph.microsoft.com/v1.0/planner/tasks/{task-id}/details,,This query requires a task id.  To find the ID of the task you can run: GET https://graph.microsoft.com/v1.0/me/planner/tasks,https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/plannertaskdetails_get,AAD,,,
```

Comment:
No, that column is deprecated and we now prompt users to change the scopes if they get an access denied message.